Firmware/OpenLog_Artemis/settings.h

```diff
@@ -8,6 +8,7 @@ typedef enum
   DEVICE_DISTANCE_VL53L1X,
   DEVICE_GPS_UBLOX,
   DEVICE_PROXIMITY_VCNL4040,
+  DEVICE_TEMPERATURE_TMP102,
```

Comment:
Please move this to the bottom of the list - above DEVICE_TOTAL_DEVICES - to preserve the existing enums.